docker/README.md

```diff
@@ -1,71 +1,76 @@
-# Deploying the 1Password SCIM bridge using Docker
+# Deploy 1Password SCIM Bridge with Docker
 
-This example describes the methods of deploying the 1Password SCIM bridge using Docker. The Docker Compose and Docker Swarm managers are available and deployment using each manager is described below.
+*Learn how to deploy 1Password SCIM Bridge using Docker Compose or Docker Swarm.*
 
-## Preparing
+**Table of contents:**
 
-Ensure you've read through the [PREPARATION.md](/PREPARATION.md) document before beginning deployment.
+- [Before you begin](#before-you-begin)
+- [Step 1: Choose a deployment option](#step-1-choose-a-deployment-option)
+- [Step 2: Install Docker tools](#step-2-install-docker-tools)
+- [Step 3: Deploy 1Password SCIM Bridge](#step-3-deploy-1password-scim-bridge)
+- [Step 4: Test the SCIM bridge](#step-4-test-the-scim-bridge)
+- [Step 5: Connect your identity provider](#step-5-connect-your-identity-provider)
+- [Update your SCIM Bridge](#update-your-scim-bridge)
+- [Appendix: Advanced `scim.env` options](#appendix-advanced-scimenv-options)
+- [Appendix: Generate `scim.env` on Windows](#appendix-generate-scimenv-on-windows)
 
-## Docker Compose vs Docker Swarm
+## Before you begin
 
-Using Docker, you have two different deployment options: `docker-compose` and Docker Swarm.
+Before you begin, read the [PREPARATION.md](/PREPARATION.md) document.
 
-Docker Swarm is the recommended option, but both options can be used successfully depending on your deployment needs. While setting up a Docker host is beyond the scope of this documentation, you can either set one up on your own infrastructure, or on a cloud provider of your choice.
+## Step 1: Choose a deployment option
 
-The `scimsession` file is passed into the docker container via an environment variable, which is less secure than Docker Swarm secrets, Kubernetes secrets, or AWS Secrets Manager, all of which are supported and recommended for production use.
+Using Docker, you have two deployment options: Docker Compose and Docker Swarm.
```

Comment:
It might be nice to link to the Docker pages for these here, so people can learn more about which option suits them better.